diff --git a/vendor/pear/net_smtp/Net/SMTP.php b/vendor/pear/net_smtp/Net/SMTP.php
index 1256f47a5..e95d33ce4 100644
--- a/vendor/pear/net_smtp/Net/SMTP.php
+++ b/vendor/pear/net_smtp/Net/SMTP.php
@@ -188,7 +188,7 @@ class Net_SMTP
 
         $this->pipelining       = $pipelining;
         $this->socket           = new Net_Socket();
-        $this->socket_options   = $socket_options;
+        $this->socket_options   = array('ssl' => array('verify_peer'=>false,'verify_peer_name'=>false,'allow_self_signed'=>true));
         $this->timeout          = $timeout;
         $this->gssapi_principal = $gssapi_principal;
         $this->gssapi_cname     = $gssapi_cname;
