diff --git a/bin/cron.php b/bin/cron.php
index 295af38a2..909365c03 100644
--- a/bin/cron.php
+++ b/bin/cron.php
@@ -9,6 +9,10 @@
  +--------------------------------------------------------------------+
  */
 
+if (!empty($_SERVER['PRESSFLOW_SETTINGS'])) {
+  ini_set('session.save_handler', 'files');
+}
+session_start();
 
 require_once '../civicrm.config.php';
 require_once 'CRM/Core/Config.php';
