diff --git a/settings_location.php b/settings_location.php
new file mode 100644
index 000000000..84f41a725
--- /dev/null
+++ b/settings_location.php
@@ -0,0 +1,12 @@
+<?php
+
+/**
+ * The directory name where civicrm.settings.php file is located.
+ * Used where CiviCRM is part of an install profile like CiviCRM Starterkit.
+ */
+if (!defined('CIVICRM_CONFDIR')) {
+  if (!empty($_ENV['PANTHEON_SITE'])) {
+    $pantheon_home = $_ENV['HOME'] . ($_ENV['HOME'] !== '/' ? '/' : '');
+    define( 'CIVICRM_CONFDIR', $pantheon_home . 'code/sites/default/' );
+  }
+}
